examples/c/sim_wt_model_nx6.c

```diff
@@ -316,13 +316,13 @@ int main()
 				gnsf_dim->nu = nu;
 				gnsf_dim->nx1= nx;
 				gnsf_dim->nx2= 0;
-				gnsf_dim->ny = nx;//nx + nu;
-				gnsf_dim->nuhat = 2;//nx + nu;
+				gnsf_dim->ny = 5;
+				gnsf_dim->nuhat = 0;
```

Comment:
What does the change mean? Why was it larger before?

Reply:
Before, I was using y=x and uhat = u; which means that phi takes the whole state and control vector as an input.
To reduce the size of the matrix vector multiplications, I reformulated the model. Using only the entries of these vectors needed to evaluate phi.
Therefore I needed to make some changes in the generating .m files, which are also included in this commit.